portality/scripts/application_status_report.py

```diff
@@ -5,7 +5,7 @@
     of applications in each status(new, accepted, rejected) for each year."""
 
 
-def date_applied_query(date_year):
+def date_applied_query(date_year_from, date_year_to,):
```

Comment:
probably worth adding the `track_total_hits` param in case the count reaches 10,000 (unlikely as that is)